.rodare.json

```
@@ -90,6 +90,11 @@
             "affiliation": "Helmholtz-Zentrum Dresden-Rossendorf",
             "name": "Ordyna, Paweł",
             "type": "Other"
+        },
+        {
+            "affiliation": "Oak Ridge National Laboratory",
+            "name": "Ganyushin, Dmitry",
+            "type": "Other"
```

@dmitry-ganyushin, do you have an ORCID? :)
```suggestion
            "orcid": "ABC",
            "type": "Other"
```

---

0000-0001-7337-2161